sites/docs/src/content/docs/guidelines/pipelines/requirements/semantic_versioning.md

```diff
@@ -6,13 +6,58 @@ menu:
     weight: 90
 ---
 
-Pipelines must be released with stable release tags.
-Releases must use GitHub releases and [keep a detailed changelog](https://keepachangelog.com/en/1.0.0/) file.
+- Pipelines must be released with stable release tags.
+- Releases must use GitHub releases and include a [detailed changelog](https://keepachangelog.com/en/1.0.0/) file.
+- The first release of a pipline should be version `1.0.0` (we strongly discourage pre-releases).
+- Release version tags must be numerical only (no `v` prefix) and should follow [semantic versioning](https://semver.org/) rules: `[major].[minor].[patch]`
 
-Release version tags must be numerical only (no `v` prefix) and should follow [semantic versioning](https://semver.org/) rules: `[major].[minor].[patch]`
+## Semantic Versioning Criteria
 
-For example, starting with with a release version `1.4.3`, bumping the version to:
+When deciding on the release type, consider the impact on users and whether they need to make changes to their existing workflows or configurations. If in doubt, opt for a higher version increment to clearly communicate the extent of changes to users.
 
-- `1.4.4` would be a patch release for minor things such as fixing bugs.
-- `1.5` would be a minor release, for example adding some new features.
-- `2.0` would correspond to the _major_ release where results would no longer be backwards compatible.
+Classify your release according to the highest applicable category in this order: **Major**, **Minor**, then **Patch**. A single match in a higher category overrides all lower categories.
+
+### Major Release (e.g., 1.4.3 to 2.0.0)
+
+Changes that are **not backwards compatible** and may require users to adjust their existing workflows or configurations.
+
+- **Breaking Changes**:
+  - Modifications to pipeline user interaction that are not backwards compatible.
+  - Renaming or deprecation of existing parameter(s).
+  - Changes to input sample sheet specifications (adding, dropping, or renaming mandatory columns).
+  - Major alterations to output formats or directory structures.
+  - Removal of previously supported features or tools.
+  - Upgrades to dependencies that introduce breaking changes.
+- **Significant Functional Changes**:
+  - Overhauls to the pipeline's core functionality or workflow.
```

Comment:
Need examples

Reply:
Let's just take this out, I think it will be covered by things in 'breaking changes'.